### ISSUE TITLE
Add predefined palettes as dropdown.

### ISSUE THREAD

**Issue by Nemecsek** [chekoduadarsh/YOPO-You-Only-Plot-Once#7]
**Is your feature request related to a problem?**
An enhancement: to give the possibility to choose a color palette from a dropdown menu. 

There are too many predefined palettes at [https://plotly.com/python/builtin-colorscales/](https://plotly.com/python/builtin-colorscales/, and the menu would become too long), so a dozen should be enough. 

As a palette list I would consider:
* Viridis
* Magma
* Turbo
* Bluered
* Hot
* RdBu
* YlOrRd
* Thermal
* Agsunset
* Balance
* Portland
* Edge

If the dropdown menu shows the palette colors (each image can be downloaded from the page) together with the name, it becomes very intuitive.

**Comment by chekoduadarsh** [comment]
@Nemecsek 

**Thank you for trying YOPO**

Yes providing the option for color pallet will help the user, but I am thinking what if it becomes part when we configure the dash board. like this

```python
dashboard(input=df,mode="inline",port=8052,  plotly_config = {theme:'dark',color_palette:'Turbo'})
```

This way user needs to click less while generating the plot, and can establish common color scheme across all the plot. If you have any objection to this approach please let me know.

**Comment by Nemecsek** [comment]
This possibility is even better. In this way the style is coherent among the plots.

**Comment by chekoduadarsh** [comment]
@Nemecsek Fixing the enhancement !! Sorry for the accidental multiple commits

**Comment by chekoduadarsh** [comment]
Fixed in 0.0.16
